managed-ledger/pom.xml

```diff
@@ -97,8 +97,8 @@
     </dependency>
 
     <dependency>
```

Comment:
This depedency could be possibly removed alltogether, no?

Reply:
I'll check that after investigating why the legacy log4j dependency was there previously.